ivc/src/ivc/interpreter.rs

```diff
@@ -887,16 +889,23 @@ where
 }
 
 /// Builds selectors for the IVC circuit.
+/// The round constants for Poseidon are not added in this function, and must be
+/// done separately.
+/// The size of the array is the total number of public values required for the
+/// IVC. Therefore, it includes the potential round constants required by
+/// the hash function.
+// FIXME: rc should be handled here or in the lens
```

Comment:
I'm working on this (slowly, there's a PR...) -- my current idea is to put it into `ColumnIndexer`, and thus the `fixed_selectors` for `IVCColumn` would call `fixed_selectors` for `PoseidonColumn`... For now this is fine, but I'd also prefer to just add `build_selectors` function for poseidon, and call it from this IVC function. Anyway, if it works it works :)